homeassistant/components/alexa/intent.py

```diff
@@ -39,6 +41,17 @@ def async_setup(hass):
     """Activate Alexa component."""
     hass.http.register_view(AlexaIntentsView)
 
+    # Resolution behavior deprecation warning
+    async_create_issue(
+        hass,
+        DOMAIN,
+        "resolution_behavior_deprecated",
+        breaks_in_ha_version="2023.7.0",
+        is_fixable=False,
+        severity=IssueSeverity.WARNING,
+        translation_key="resolution_behavior_deprecated",
+    )
+
```

Comment:
I suggest we move the deprecation part to a new PR and leave this one for the nearest resolutions

Reply:
Fair enough. For now I'd only exose the new ID Variable which is the most essential.
The Docs PR is already changed for that. Because it's a bad Idea to introduce new variables just to remove them later anyway. The ID Variable however is made to stay there in the long run.

Reply:
Should be fine now.